### ISSUE TITLE
Provide a way to restore files on a different platform

### ISSUE THREAD

**Issue by BrenBarn** [duplicati/duplicati#3910]
There was discussion of this some time ago on the forum (https://forum.duplicati.com/t/cross-platform-restore-not-possible/4520/2) but I don't see any issue was ever created about it.

Based on the discussion there it seems that the obstacle to cross-platform restoring is mainly stuff like different file attributes and path separators.  It would seem a shame to have a large backup with all the actual data intact, but be unable to restore just due to these issues.  As mentioned in that thread, it ought to be possible to do some kind of path translation and perhaps just ignore the metadata (i.e., use default values for attributes or something).

In short, it would be good to have a supported way to back up files on one platform and restore them on another.

**Comment by RichieFrame** [comment]
Not just path separators, but also file names. On Linux for example, the default filesystems are case sensitive, so Image.jpg and IMAGE.JPG would be 2 unique files, but if you restore a backup set containing them to a windows machine, the file location is the same, and one would overwrite the other

**Comment by ts678** [comment]
I'm not sure that comment is correct. It is for backups, but restores might be allowed, per the following:

https://github.com/duplicati/duplicati/blob/99d0cb3c57b309f02672e22989d58d1058a16d4c/Duplicati/Server/webroot/ngax/scripts/controllers/RestoreController.js#L307-L318

Care to test?
